content/faq.md

```diff
@@ -1,9 +1,29 @@
-title: complate Frequently Asked Questions 
+title: complate Frequently Asked Questions
 h1: FAQ
-subtitle: Frequently asked questions
+subtitle: Frequently Asked Questions
 safe: false
 
 
+Is complate Right for Me? <span id="trade-offs"></span>
```

Comment:
This is a bit of a tacky title - sort of on purpose, but if anyone has any alternative suggestions, I'm all ears.